extension/ts/extensions/banner.ts

```diff
@@ -16,17 +23,21 @@ const setLogo = (id: string) =>
 const setCSS = (id: string, css: Partial<CSSStyleDeclaration>) =>
 	editElement(id, (element) => Object.entries(css).forEach(([key, value]) => (element.style[key] = value)));
 
-const setFavicon = () => {
-	const element = Array.from(document.getElementsByTagName("link")).find(
-		(element) => element.rel === "icon" && element.type === "image/x-icon"
-	);
-	if (element) {
-		element.href = chrome.runtime.getURL("img/favicon.ico");
-	}
-};
+const setFavicon = () =>
+	Array.from(document.getElementsByTagName("link"))
+		.filter((element) => element.rel === "icon")
+		.map((element) => {
+			element.href = chrome.runtime.getURL("img/favicon.ico");
+			element.type = "image/x-icon";
```

Comment:
we expect this to only ever be a 1 element array after the filtering step right

Reply:
No  It's possible to have multiple (which library thing sometimes does and it exposed a bug)